### DISCUSSION TITLE
input type of file  and  datepicker

### DISCUSSION THREAD

**Discussion by iliassela** [myndpm/open-source#32] [Dynamic Forms]
input type of file don't work and also datepicker don't work can i have i exemple of that plz ??

**Comment by matheo** [comment]
For the **DATEPICKER**:
I see it working in the website: https://mynd.dev/demos/dyn-forms/stepper-form/step-1
so it's possibly related to the required modules to be imported to make it work.

This PR shows the code required and it seems that you need to provide your DateAdapter of preference:
https://github.com/myndpm/open-source/pull/19/files#diff-c6236301c3df2187bc3f5a9306d800af4af02930743fda8403843fdd155b96f7

I hope it fixes any exception related to missing DateAdapters.

**Reply by omryoz** [reply]
hi, how can i set the date format ? i.e DD-MM-YYYY ?

**Comment by matheo** [comment]
About **INPUT** with type file:
I haven't played with it yet and I've excluded that type from the input it seems,
it might need some considerations to correctly send the form to the Backend, like the form-encoding headers.

Also, we could build a custom control dedicated to it like https://stackoverflow.com/questions/31867194/file-upload-with-angular-material
